corehq/apps/linked_domain/views.py

```diff
@@ -33,7 +33,7 @@ def get_latest_released_app_source(request, domain, app_id):
     if master_app.domain != domain:
         raise Http404
 
-    requester = request.GET.get('requester')
+    requester = request.META.get('HTTP_HQ_REMOTE_REQUESTER', None)
```

Comment:
expect a header instead of a get param to match https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/linked_domain/decorators.py#L11